### PR TITLE
BLM: added optimization feature for umbral ice transpose lines

### PR DESCRIPTION
**Pull request by lhn1703**
Niche high end optimization to squeeze out 84 extra potency in niche transpose lines.

### CODE REVIEW

File: XIVComboExpanded/CustomComboPreset.cs [hunk 1]
```diff
@@ -256,6 +256,13 @@ public enum CustomComboPreset
     [CustomComboInfo("Enochian No Sync Feature", "Fire 4 and Blizzard 4 will not sync to Fire 1 and Blizzard 1.", BLM.JobID)]
     BlackEnochianNoSyncFeature = 2518,
 
+    [IconsCombo([BLM.Fire4, BLM.Blizzard4, UTL.ArrowLeft, ADV.Swiftcast, BLM.Triplecast, BLM.Blizzard3, UTL.Blank, BLM.Blizzard, UTL.Blank, UTL.Idea])]
+    [SectionCombo("Single Target")]
+    [ParentCombo(BlackEnochianFeature)]
+    [SecretCustomCombo]
+    [CustomComboInfo("Enochian Umbral Ice Feature", "Replace Fire 4 and Blizzard 4 with Instant-Blizzard 3 or Blizzard 1 when in Umbral Ice 1 or 2. This facilitates Umbral Ice transpose line (Astral Fire -> transpose -> instant-B3; negating 0.7x Blizzard 3 damage multiplier in Astral Fire 3) and GIRL [Geiji Ice Recovery Line] emergency recovery transpose line (Astral Fire -> Tranpose -> Paradox -> Blizzard 1).", BLM.JobID)]
```
Comment (MKhayle) [inline XIVComboExpanded/CustomComboPreset.cs]: Geiji Ice Recovery Line.

Comment (MKhayle) [reply XIVComboExpanded/CustomComboPreset.cs]: Why can't you be normal, BLM?

Comment (lhn1703) [reply XIVComboExpanded/CustomComboPreset.cs]: Black mages are wide as our ley lines, but deep as the void.

Comment (kaedys) [reply XIVComboExpanded/CustomComboPreset.cs]: > Why can't you be normal, BLM?

☝️☝️☝️

**Comment by MKhayle** [comment]
even more nerd options for BLM 🥇

**Pull request merged**